examples/js/exporters/GLTFExporter.js

```diff
@@ -53,20 +60,26 @@ THREE.GLTFExporter.prototype = {
 	 * @param  {THREE.Scene or [THREE.Scenes]} input   THREE.Scene or Array of THREE.Scenes
 	 * @param  {Function} onDone  Callback on completed
 	 * @param  {Object} options options
-	 *                          trs: Exports position, rotation and scale instead of matrix
-	 *                          binary: Exports `.glb` as ArrayBuffer, instead of `.gltf` as JSON
 	 */
 	parse: function ( input, onDone, options ) {
 
 		var DEFAULT_OPTIONS = {
 			trs: false,
 			onlyVisible: true,
 			truncateDrawRange: true,
-			embedImages: true
+			embedImages: true,
+			animations: []
 		};
 
 		options = Object.assign( {}, DEFAULT_OPTIONS, options );
 
+		if ( options.animations.length > 0 ) {
+
+			// Only TRS properties, and not matrices, may be targeted by animation.
+			options.trs = true;
```

Comment:
Is this mandatory by the spec? Or is it just to ease the loader. I understand that the samplers for animation should be just `translate`, `rotate and `scale`, but if the node has a initial position defined by a matrix should that affect anyhow?

Reply:
It's required by the spec, not sure why: https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#transformations

Reply:
Ah ok I didn't notice that comment. thanks. I'm also not sure why is needed, but that's another conversation :) Nice job here!